### PR TITLE
refactor(trackerless-network): NET-873 move `src/proto/` to `generated/`

### PR DESCRIPTION
**Pull request by juslesan**
## Summary

moved generated proto files out of src

blocking #2820 

## Other changes

regenerated DHT proto files in the trackerless-network

### CODE REVIEW

File: packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts [hunk 1]
```diff
@@ -336,6 +337,10 @@ export interface IConnectionLockRpcClient {
      * @generated from protobuf rpc: gracefulDisconnect(dht.DisconnectNotice) returns (google.protobuf.Empty);
      */
     gracefulDisconnect(input: DisconnectNotice, options?: RpcOptions): UnaryCall<DisconnectNotice, Empty>;
+    /**
+     * @generated from protobuf rpc: setPrivate(dht.SetPrivateRequest) returns (google.protobuf.Empty);
+     */
+    setPrivate(input: SetPrivateRequest, options?: RpcOptions): UnaryCall<SetPrivateRequest, Empty>;
```
Comment (teogeb) [inline packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts:343]: These changes are not part of this PR?

**Review by teogeb — CHANGES_REQUESTED** [review]

File: packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts (hunk 1 above)
Comment (juslesan) [reply packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts:343]: These are unused parts of the generated DHT proto files in the trackerless network

Comment (teogeb) [reply packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts:343]: As this PR is about moving files, could revert these changes in this PR.

Comment (juslesan) [reply packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts:343]: Added as other changes in the description

Comment (juslesan) [reply packages/trackerless-network/generated/packages/dht/protos/DhtRpc.client.ts:343]: Made a ticket about removing unused generated code from the higher layers

**Review by teogeb — APPROVED** [review]
